pkgs/by-name/ev/everforest-cursors/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "everforest-cursors";
+  version = "3212590527";
+
+  src = fetchurl {
+    url = "https://github.com/talwat/everforest-cursors/releases/download/${version}/everforest-cursors-variants.tar.bz2";
+    hash = "sha256-xXgtN9wbjbrGLUGYymMEGug9xEs9y44mq18yZVdbiuU=";
+  };
+
+  sourceRoot = ".";
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/share/icons
+    cp -r ./everforest-cursors* $out/share/icons
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
remove with lib; as it increases eval times

Reply:
Done